http/router.go

```diff
@@ -48,7 +48,7 @@ func (p *LnurlPayInfoPayload) ToNotification(query *MobilePushWebHookQuery) *not
 type LnurlPayInvoicePayload struct {
 	Template string `json:"template" binding:"required,eq=lnurlpay_invoice"`
 	Data     struct {
-		Amount   string `json:"amount" binding:"required"`
+		Amount   int    `json:"amount" binding:"required"`
```

Comment:
Should we validate the amount here, or should the validation take place in the application/SDK? I've seen the breez-lnurl server parses a uint, but I would say breez-lnurl and notify should not be dependent on each other to validate data, especially when vendors have differing implementations.
```suggestion
		Amount   int    `json:"amount" binding:"required,min=1"`
```

Reply:
Good point, fixed.